mkosi/resources/reproducible.debian.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
```

Comment:
I've preferred this style a long time as well, but I think this has the downside of potentially spamming output with stuff from `/etc/profile.d`.
```suggestion
#!/bin/bash
set -x
```

Reply:
I was suppose to remove `-x`. But there is no output from this (strangely enough) so I forgot about it.